package.json

```diff
@@ -67,8 +67,8 @@
     "eslint-plugin-react": "7.20.5",
     "eslint-plugin-standard": "4.0.1",
     "husky": "4.2.5",
+    "lint-staged": "10.2.11",
```

Comment:
IIUC, we don't use that now, right? Because you just build our own `lint-staged` command.

Reply:
err no I don't, I just tell it where to look i thought, but don't worry, i'll read the docs properly and i'll come back to this after I've got some issues fixed